src/solvers/smt2_incremental/smt_response_validation.h

```diff
@@ -3,11 +3,14 @@
 #ifndef CPROVER_SOLVERS_SMT2_INCREMENTAL_SMT_RESPONSE_VALIDATION_H
 #define CPROVER_SOLVERS_SMT2_INCREMENTAL_SMT_RESPONSE_VALIDATION_H
 
+#include <util/irep.h>
 #include <util/nodiscard.h>
 
-#include <solvers/smt2_incremental/ast/smt_responses.h>
```

Comment:
`response_or_errort` handles the type it is specialised with by-value like optional does. `response_or_errort` is specialised with `smt_responset` for the return value of `validate_smt_response`. Therefore I would say that it is incorrect to remove this include as it is used.

Reply:
I have now added `// IWYU pragma: keep` instead.